src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -53,7 +53,7 @@ class PressableWithSecondaryInteraction extends Component {
 
         // On Web, Text does not support LongPress events thus manage inline mode with styling instead of using Text.
         return (
-            <Pressable
+            <TouchableOpacity
```

Comment:
We need this to be a `TouchableOpacity` for the ReportActions. It is not a "new" one, we are just moving it around. Once we decide how we want to handle those, then we'll replace it here.

Reply:
@0xmiroslav does this blurb from the PR description answer your question?

>Surrounding the existing TouchableOpacity with a PressableWithSecondaryInteraction component did not work on mobile, as the triggered touch event was the TouchableOpacity one, so I replaced the TouchableOpacity with the PressableWithSecondaryInteraction and gave it opacity capabilities. All working nice now!

Reply:
ReportActions currently use a TouchableOpacity, and the current UI is based on that. I'm moving that TouchableOpacity to the PressableWithSecondaryInteraction component (which currently surrounds the TouchableOpacity for ReportActions) so that it works correctly in all cases. Using Pressable here would mess the UI.

We are currently designing/developing a new component to replace TouchableOpacity. When that happens, it'll simply get replaced here too.